src/Atmos/Model/AtmosModel.jl

```diff
@@ -185,6 +194,13 @@ function diffusive!(m::AtmosModel, diffusive::Vars, ∇transform::Grad, state::V
   # momentum flux tensor
   diffusive.ρτ = scaled_momentum_flux_tensor(m.turbulence, ρν, S)
 
+  # turbulent Prandtl number
+  diag_ρν = ρν isa Real ? ρν : diag(ρν) # either a scalar or matrix
+  # Diffusivity D_t = ρν/Prandtl_turb
+  D_T = diag_ρν * inv_Pr_turb
```

Comment:
While you're doing this, lets rename this to `ρD_t`.

Reply:
Makes sense. I love this naming scheme!